src/deploy/firestore/deploy.ts

```diff
@@ -1,10 +1,41 @@
 import * as clc from "colorette";
 
 import { FirestoreApi } from "../../firestore/api";
+import * as types from "../../firestore/api-types";
 import { logger } from "../../logger";
 import * as utils from "../../utils";
 import { RulesDeploy, RulesetServiceType } from "../../rulesDeploy";
 import { IndexContext } from "./prepare";
+import { FirestoreConfig } from "../../firebaseConfig";
+import { sleep } from "../../utils";
+
+async function createDatabase(context: any, options: any): Promise<void> {
```

Comment:
Nit: Should use Options (from options.ts) here instead of any - that should clear out a bunch of these type warnings.

Reply:
Done